spec/hatchet/python_spec.rb

```diff
@@ -0,0 +1,10 @@
+require_relative '../spec_helper'
+
+describe "Python!!!!!!!!!!!" do
+  it "🐍" do
+    Hatchet::Runner.new('python-getting-started', stack: DEFAULT_STACK).deploy do |app|
```

Comment:
@schneems can check me here, but I don't think you need to explicitly specify the `stack` here

Reply:
Correct, but explicit > implicit for python. I think my intention here was to declare which tests should be stack independent so that they can all be changed with one variable change.